client/my-sites/subscribers/components/subscriber-data-views/subscriber-data-views.tsx

```diff
@@ -351,9 +351,11 @@ const SubscriberDataViews = ( {
 		setSearchTerm( currentView.search ?? '' );
 
 		// Handle filter option from the view.
-		setFilterOption(
-			( currentView.filters?.[ 0 ]?.value as SubscribersFilterBy ) ?? SubscribersFilterBy.All
-		);
+		const filterValues =
+			( currentView.filters
+				?.filter( ( filter ) => filter.value !== undefined )
```

Comment:
This `undefined` check is needed to prevent undefined values from triggering an api call.